rules/typescript.js

```diff
@@ -167,6 +167,13 @@ module.exports = {
         selector: 'variable',
       },
     ],
+    /**
+     * TODO [@typescript-eslint/eslint-plugin@>=4.7.0] (@ridedott/platform)
+     * Somehow this rule is reported before using current version (4.6.1)
+     * enable it when upgrading to V4.7.0, see
+     * https://github.com/typescript-eslint/typescript-eslint/releases/tag/v4.7.0.
+     */
```

Comment:
@mad-it added a TODO to not forget.

Reply:
But its part of eslint/parser https://github.com/ridedott/eslint-config/pull/688. Don't really understand why you want to disable it, just to enable it in the next minor version upgrade.